documentation/emacs.md

```diff
@@ -11,7 +11,7 @@ image:
 ads: false
 ---
 
-This document assumes that you have installed Lean and Emacs following the instructions on the [Download](../../download) page.
+This document assumes that you have installed Lean and the Lean Emacs packages following the instructions on the [Download](../../download) page and the [lean-mode](https://github.com/leanprover/lean-mode) readme, respectively.
 
 ## About Emacs
```

Comment:
@avigad This file contains some outdated information as well as duplicate content from https://github.com/leanprover/lean-mode/blob/master/README.md. Should we try to consolidate that?

Reply:
@Kha In fact, there is another open pull request to this repo which tries to clean up some of the outdated documentation: https://github.com/leanprover/leanprover.github.io/pull/70

These two PR's conflict, since that one deletes `emacs.md`.

Reply:
Ah, I didn't know that you changed this file as well